package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@productboard/slate-edit-list",
   "description": "A Slate plugin to handle keyboard events in lists.",
-  "version": "0.17.0",
+  "version": "0.18.0",
```

Comment:
pls don't upgrade the version manually. It's better to do it with [npm version](https://docs.npmjs.com/cli/version)

Reply:
pls revert